samples/tab-adaptive-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,123 @@
+# Tab With AdpativeCard Flow
+
+This App talks about the Teams tab which displays Adaptive card with Node JS
+
+This bot has been created using [Bot Framework v4](https://dev.botframework.com), it shows how to create a simple bot that accepts Adaptive Cards V1.4 to render in Teams tab.
+
+This feature shown in this sample is in Public Developer Preview and is supported in desktop and mobile.
+
+## Key features
+
+- Tab showing Adaptive card with action controls
+
+![Home Page](Images/home-page.png)
+
+- Task module
+
+![Task Module](Images/task-module.png)
+
+- Sign in card
+
+![Sign in Card](Images/sign-in.png)
+
+- Sign in popup
+
+![Sing in popup](Images/sign-in-popup.png)
+
+- Sign out card
+
+![Sign out Card](Images/sign-out.png)
+
```

Comment:
Could you please aslo add screenshot of "More" Tab?

Reply:
removed this as this contains the same content